test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4311,18 +4313,22 @@ def _load_configuration(self, infile, objtype=None):
                             self.logger.error("Failed to restore "
                                               + "configuration: %s" % k)
                             return False
-                    finally:
-                        if os.path.isfile(fn):
-                            self.du.rm(path=fn, force=True, sudo=True)
+                        self.du.rm(path=fn, force=True, sudo=True)
+                    else:
+                        self.logger.error("Failed to restore "
+                                          + "configuration: %s" % k)
+                        return False
                 return True
             elif objtype == MGR_OBJ_NODE:
-                conf = sconf[str(objtype)]
-                nconf = {}
                 nconf = conf[str(self.hostname)]
                 for k, v in nconf.items():
-                    try:
-                        fn = self.du.create_temp_file()
-                        self.du.chmod(path=fn, mode=0o644)
```

Comment:
Thanks @saritakh for making the changes. one suggestion, I see that if we remove the `try` block, there might be exceptions at places like `fd.write`. How about we keep the `try...finally` block as it was and add an `except` block which shall `return False`? Rest looks good to me.